third_party/protobuf/src/google/protobuf/stubs/once.cc

```diff
@@ -39,9 +39,7 @@
 
 #ifndef GOOGLE_PROTOBUF_NO_THREAD_SAFETY
 
-#if defined(STARBOARD)
-#include "starboard/thread.h"
-#elif defined(_WIN32)
+#if defined(_WIN32)
```

Comment:
I _think_ this is a builtin macro on windows, are you okay with having this be different across platforms?

Reply:
This is for the host toolchain